libs/libcups/patches/100-components.patch

```diff
@@ -0,0 +1,42 @@
+--- a/configure
++++ b/configure
+@@ -1598,6 +1598,7 @@ Optional Packages:
+   --with-operkey          set the default operator @AUTHKEY value
+   --with-components       set components to build:
+ 			    - "all" (default) builds everything
++			    - "embedded" builds everything except man and notifier
```

Comment:
Do we need doc and examples or does things break badly if you remove those?

Reply:
hplip has a hard dependency on libcups as both cups driver and sane driver uses cups. hplip-cups is not useful for OpenWrt as cups is not available anymore. I patched hplip-sane do get rid of cups dependency (it was only used to detect hp scanners using cups, which does not make sense for OpenWrt). However, I wasn't able to patch hplip to build without cups. So, I'm building cups only for libs and headers. It's a build-only dependency.

Now I'm using --with-components=core for ./configure, but nothing will change at all as I'm using a custom Build/Compile to build only what I need. In fact, I got rid of all patches.